com.unity.render-pipelines.high-definition/Documentation~/Customizing-HDRP-materials-with-Shader-Graph.md

```diff
@@ -66,6 +66,23 @@ You should only expose a property if you intend to change it in the Inspector wh
 
 If a property is not exposed, you can still edit it. To edit the property for every Material instance that uses the selected Shader, open the Shader Graph Asset and edit the property directly in the Blackboard. To edit the property for a single Material instance that uses the Shader, use a script. 
 
+**:warning: HDRP reserves a set of property names that you should not use in the Blackboard. If you use any of them, the Material will most likely not work correctly. The restricted property names in this table:**
```

Comment:
For the list of restricted property names and their uses, see the following table: